code/src/styles/SubmitButton.css

```diff
@@ -0,0 +1,16 @@
+@import url('https://fonts.googleapis.com/css2?family=Trispace&display=swap');
+
+* {
+  font-family: 'Trispace', sans-serif;
+  text-rendering: optimizeLegibility;
+}
+
+.submit-button {
+    border-radius: 20px;
+    padding: 5px;
+    background-color: #ffadad;
+    border: 1px solid #ffadad; 
```

Comment:
Do you need this border since it has the same color as the background?  Or could you just type `border:none;`?